src/logging.cpp

```diff
@@ -160,6 +160,7 @@ const CLogCategoryDesc LogCategories[] =
     {BCLog::I2P, "i2p"},
     {BCLog::IPC, "ipc"},
     {BCLog::LOCK, "lock"},
+    {BCLog::UTIL, "util"},
```

Comment:
Ok, yes, adding a util category might sense. We might want to migrate at least "rand" into here as well, at some point (not in this PR though).